assets/l10n/en-US.ftl

```diff
@@ -534,6 +538,7 @@ label_was_added = {$who} was added
 label_was_removed = {$who} was removed
 label_you = You
 label_you_were_added_to_group = You were added to the group
+label_you_were_added_to_group_by_user = {$user} added you to the group
```

Comment:
`label_you_were_added_to_group_by_user` -> `label_user_added_you_to_group`
`{$user} added you to the group` -> `{$user} added you`